internal/handlers/common/msg_debugerror.go

```diff
@@ -34,10 +37,29 @@ func MsgDebugError(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
 		return nil, err
 	}
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/640
+	// check if parameter is an error code
+	if n, err := strconv.Atoi(expected); err == nil {
+		errCode := ErrorCode(n)
```

Comment:
What if `n` is something that is not present in the list of `ErrorCode`s? 
For example, if `n == 123456`, how would this work?

Reply:
Good thing to cover with a dance test

Reply:
@rumyantseva The ErrorCode.String() method will return it as `"ErrorCode(123456)"`.